examples/bricks/WalletBrick/5-ValueProp-Customization.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import Wallet from '../../../src/bricks/wallet';
+
+import initMercadoPago from '../../../src/mercadoPago/initMercadoPago';
+
+initMercadoPago('TEST-f4563544-ce69-40c3-b88e-6e7d1bd93a83');
+
+const ExampleVisualCustomizationWalletBrick = () => {
```

Comment:
Poderíamos deixar mais claro que se trata de uma customização de value prop no nome da função?
```suggestion
const ExampleValuePropCustomizationWalletBrick = () => {
```